src/utils/IdealGraphVisualizer/README.md

```diff
@@ -33,7 +33,7 @@ Ideal graphs are dumped at the following points:
 * `N=2`: additionally, after every major phase
 * `N=3`: additionally, after every minor phase
 * `N=4`: additionally, after every loop optimization
-* `N=5`: additionally, after every effective IGVN and every macro expansion step (slow)
+* `N=5`: additionally, after every effective IGVN, macro elimination, and expansion step (slow)
```

Comment:
Typo `,` before `, and`.

Reply:
@sarannat: I also think you should write this out in full: `... macro elimination, and macro expansion step (slow)`. The `,` before `and` in itself is not incorrect, but a question of style (see [Oxford comma](https://en.wikipedia.org/wiki/Serial_comma)).